vmdb/app/models/vm_scan.rb

```diff
@@ -199,7 +226,13 @@ def call_snapshot_delete
       if vm.ext_management_system
         $log.info("MIQ(scan-action-call_snapshot_delete) Deleting snapshot: reference: [#{mor}]")
         begin
-          delete_snapshot(mor)
+          # TODO: should this logic be moved to a VM subclass implementation?
+          #       or, make type-specific Job classes.
+          if vm.kind_of?(VmOpenstack)
+            vm.ext_management_system.vm_delete_evm_snapshot(vm, mor)
```

Comment:
Yes, the TODO comment above states that. This is a general issue for vm_* methods defined in ems classes. This is something we should address in a separate PR.